src/spec_tools/json_schema.cljc

```diff
@@ -202,7 +202,10 @@
 (defmethod accept-spec 'clojure.spec/and [dispatch spec children]
   (simplify-all-of {:allOf children}))
 
-; merge
+(defmethod accept-spec 'clojure.spec/merge [dispatch spec children]
+  {:type "object"
+   :properties (apply merge (map :properties children))
+   :required (apply concat (map :required children))})
```

Comment:
What if the latter children defines a optional key which is required in first children? Currently the key would be included in required list. Also, if the same key is in multiple maps, it will be present multiple times in required list. Not sure whan JSON Schema says about that.